.github/workflows/build_and_push_nf_base_images.yml

```diff
@@ -0,0 +1,68 @@
+name: Push new nextflow images to ECR
```

Comment:
> New Features
Add new GH workflow that automatically pushes new base images to public ECR

The wf is specific to nextflow base images. Can you clarify that in your release notes